internal/armada/server/job_validation.go

```diff
@@ -11,7 +11,7 @@ func validateJobsCanBeScheduled(jobs []*api.Job, allClusterSchedulingInfo map[st
 	activeClusterSchedulingInfo := scheduling.FilterActiveClusterSchedulingInfoReports(allClusterSchedulingInfo)
 	for i, job := range jobs {
 		if !scheduling.MatchSchedulingRequirementsOnAnyCluster(job, activeClusterSchedulingInfo) {
-			return fmt.Errorf("job with index %d is not schedulable on any cluster", i)
+			return fmt.Errorf("[validateJobsCanBeScheduled] the %d-th job (job ID %s) can't be scheduled on any cluster", i, job.Id)
```

Comment:
I like this

Reply:
:)

Reply:
Nitpicks - This will create 1-th, 2-th and we typically use `job id or job Id`

Reply:
I updated the message to be `fmt.Errorf("[validateJobsCanBeScheduled] job %d out of %d with Id %s can't be scheduled on any cluster", i, len(jobs), job.Id)`.